tests/codegen/llvm/map_key_string.ll

```diff
@@ -30,15 +30,16 @@ entry:
   call void @llvm.memcpy.p0i8.p0i8.i64(i8* align 1 %7, i8* align 1 %8, i64 64, i1 false)
   %9 = bitcast [64 x i8]* %str1 to i8*
   call void @llvm.lifetime.end.p0i8(i64 -1, i8* %9)
-  %10 = bitcast i64* %"@x_val" to i8*
-  call void @llvm.lifetime.start.p0i8(i64 -1, i8* %10)
-  store i64 44, i64* %"@x_val"
+  %10 = sext [64 x i8]* %str1 to i64
```

Comment:
this bit's become substantially different; originally we stored a constant `44`, now we are storing something computed from `%str1`..